assets/server/admin/realms/index.html

```diff
@@ -45,17 +45,20 @@
           <thead>
             <tr>
               <th scope="col" width="50" class="text-center">ID</th>
+              <th scope="col" width="100" class="text-center">Region</th>
               <th scope="col">Name</th>
+              <th scope="col" width="75" class="text-center">Chaff (7d)</th>                          
+              <th scope="col" width="100" class="text-center">Abuse Modeling</th> 
+              <th scope="col" width="75" class="text-center">Auth SMS</th>
               <th scope="col" width="90" class="text-center">User Report</th>
-              <th scope="col" width="75" class="text-center">Chaff</th>
-              <th scope="col" width="100" class="text-center">Region</th>
-              <th scope="col" width="120" class="text-center">Signing key</th>
+              <th scope="col" width="90" class="text-center">Signing key</th>
```

Comment:
Can we collapse some of these columns on smaller screens? Otherwise it's unusable on mobile:

![server verification apollo-project org_admin_realms(iPhone X)](https://user-images.githubusercontent.com/408570/119054743-31ff9b00-b996-11eb-98dc-9ea6d9afb269.png)

Reply:
done + also added this info to detail page so that it can be seen on mobile